.circleci/config.yml

```diff
@@ -185,11 +185,24 @@ jobs:
             errors=$(jq '.["results"] | .[] | unique_by(.["type"]) | map(select(.["type"] == "error")) | .[]' planet4-docker-compose/artifacts/pa11y/pa11y-ci-results.json)
             if [ ! -z "${errors}" ]; then echo "Errors found, see report in artifacts." && exit 1; else echo "No errors, report available in artifacts."; fi
 
-  rips-test:
+  create-release-zip:
     docker:
-      - image: rips/rips-cli:3
-    environment:
-      RIPS_APPLICATION_ID: 21
+      - image: mhart/alpine-node
     steps:
       - checkout
-      - run: rips-cli -vvv rips:scan:start -a $RIPS_APPLICATION_ID -p ~/project/ -t 1
+      - run: apk add curl zip git jq
+      - run: git submodule init && git submodule update
+      - run: npm ci
+      - run: npm run build
+      - run: rm -rf .circleci .git .githooks assets/src bin tests
+      - run: mkdir -p /tmp/workspace/
+      - run: zip -r /tmp/workspace/planet4-master-theme.zip . -x "node_modules/*"
+      - run: ls -lh /tmp/workspace/planet4-master-theme.zip
+      - run: curl https://api.github.com/repos/greenpeace/planet4-master-theme/releases/tags/${CIRCLE_TAG} | jq -r .upload_url | sed 's/{.*}/?name=planet4-master-theme/' > /tmp/workspace/upload_url
+      - run: cat /tmp/workspace/upload_url
```

Comment:
If you want to make these steps less cryptic on CircleCI UI you can break them down to `name` and `command`.
https://circleci.com/docs/2.0/configuration-reference/#run